ignore-whitespaces.js

```diff
@@ -1,15 +1,27 @@
-if (
-  /^https:\/\/gitlab\.com/.test(window.location.href) &&
-  !/\?.*w=1/.test(window.location.href)
-) {
-  var str = "";
-  if (/\?(.+=.*&)*$/.test(window.location.href)) {
-    str = "";
-  } else if (/\?(.+=.*&)*.+=.*[^&]$/.test(window.location.href)) {
-    str = "&";
-  } else {
-    str = "?";
-  }
+var isRefreshing = false;
+
+function doIt() {
+  if (
+    !isRefreshing &&
+    /^https:\/\/gitlab\.com.*\/diffs/.test(window.location.href) &&
+    !/\?.*w=1/.test(window.location.href)
+  ) {
+    var str = "";
+    if (/\?(.+=.*&)*$/.test(window.location.href)) {
+      str = "";
+    } else if (/\?(.+=.*&)*.+=.*[^&]$/.test(window.location.href)) {
+      str = "&";
+    } else {
+      str = "?";
+    }
 
-  window.location = window.location.toString() + str + "w=1";
+    window.location = window.location.toString() + str + "w=1";
+    isRefreshing = true;
+    setTimeout(function() {
+      isRefreshing = false;
+    }, 2000);
+  }
 }
+
+doIt();
+setInterval(doIt, 200);
```

Comment:
Are you sure it can't be done any other way than having run this function five times a second?

Reply:
There surely is a way cleaner solution. I tried to find something and gave up after 30 minutes.

And this works quite well ...

I will try to come up with something better next week.